charts/fiberplane-daemon/values.yaml

```diff
@@ -0,0 +1,43 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: fiberplane/fpd
+  pullPolicy: IfNotPresent
```

Comment:
Think we should set this to `Always`. Since we are using the `v2` tag (right). That way the user will get updates once they restart.